client/components/Editor/plugins/onChange.ts

```diff
@@ -71,7 +101,7 @@ const getMenuItems = (editorView) => {
 			return false;
 		}
 		const $from = editorView.state.selection.$from;
-		const selectedNode = editorView.state.selection.node;
+		const selectedNode = (editorView.state.selection as any).node;
```

Comment:
I need to figure out the best strategy here since the prosemirror type definitions do not include a `node` property on `Selection<any>`. Any thoughts @idreyn ?

Reply:
A selection of a node is a `NodeSelection` — you could cast to that type or write a type guard.